pages/index.html

```diff
@@ -4,9 +4,37 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Michelle's Portfolio Website</title>
+  <link href="/styles/style.css" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/3e8fd03e4e.js"
+  crossorigin="anonymous"></script>
 </head>
+
 <body>
-  
+<content id="main">
+<header>
+  <iframe width="0" height="0" src="https://www.youtube.com/embed/Atp8GPADPDM?controls=0?rel=0&autoplay=1" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+<nav>
+  <h1>Welcome to Michelle's Portfolio!</h1>
+  <ul class="top-bottom-links">
+    <li>Main</li>
+    <li><a href="/pages/about.html">About</a></li>
+    <li><a href="/pages/portfolio.html">Portfolio</a></li>
+  </ul>
+</nav>
+</header>
+<section id="main">
```

Comment:
id attributes must be unique across the entire page. The main id here collides with the main id assigned to the content tag above. Consider using the main tag instead (section should usually have a heading).

Running your html through a validator would catch this.

Reply:
The content here appears empty, while in the browser, there's a background image displayed. In this case, the image really _is_ the content, and would be better represented with an img tag, rather than a background style. The fact that there is actually content here is not visible to screen readers or other assistive technologies, where as using an img tag would allow (require) an alt attribute in which you could describe that there was an aurora being displayed here.